shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt

```diff
@@ -40,6 +40,17 @@ class MealApi {
         })
     }
 
+    suspend fun getSortedMeals(userLat: Double, userLon: Double, distanceUnit: Int): List<Meal> {
+        var meals: List<Meal> = getAllMeals()
+        for (meal in meals) {
+            meal.distance = locationUtil.getDistance(
+                userLat, userLon, meal.locationLat.toDouble(), meal.locationLong.toDouble(), distanceUnit
+            )
+        }
+        // TODO: Filter meals that have expired here
```

Comment:
Put this as a TODO as finding a suitable DateTime library for KMM was proving a pain in the arse.
Will tackle that next - might even pass the Date up from ios/android and use that as a temporary solution? 
Let me know your thoughts